packages/ldap-auth-backend/README.md

```diff
@@ -1 +1,194 @@
 # @immobiliarelabs/backstage-plugin-ldap-auth-backend
+
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier?style=flat-square)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
+![license](https://img.shields.io/github/license/immobiliare/backstage-plugin-ldap-auth?style=flat-square)
+![npm (scoped)](https://img.shields.io/npm/v/@immobiliarelabs/backstage-plugin-ldap-auth-backend?style=flat-square)
+
+> Customizable Authentication backend provider for LDAP servers for your [Backstage](https://backstage.io/) deployment
+
+Works either on simple stand-alone process or scaled infrastracture spanning multiple deployments using the shared PostgreSQL instance that Backstage already uses!
+
+This plugin is not meant to be used alone but with in pair with:
+
+-   The official [@backstage/plugin-catalog-backend-module-ldap](https://www.npmjs.com/package/@backstage/plugin-catalog-backend-module-ldap) which keeps in sync your LDAP users with Backstage user catalogs!
+-   Its sibling frontend package [@immobiliarelabs/backstage-plugin-ldap-auth](https://www.npmjs.com/package/@immobiliarelabs/backstage-plugin-ldap-auth)
+
+Supports Node.js `>=14.0.0`
+
+## Table of Content
+
+<!-- toc -->
+
+-   [Installation](#installation)
+-   [Configurations](#configurations)
+    -   [Connection Configuration](#connection-configuration)
+    -   [Setup Backstage official LDAP plugin](#setup-backstage-official-ldap-plugin)
+    -   [Add authentication backend](#add-authentication-backend)
+    -   [Add the login form](#add-the-login-form)
+-   [Powered Apps](#powered-apps)
+-   [Support & Contribute](#support--contribute)
+-   [License](#license)
+
+<!-- tocstop -->
+
+## Installation
+
+> These packages are available on npm.
+
+You can install them in your backstage installation with `npm workspace`
+
+```bash
+# install backend plugin
+$ npm i @immobiliarelabs/backstage-plugin-ldap-auth-backend -w backend
+# install frontend plugin
+$ npm i @immobiliarelabs/backstage-plugin-ldap-auth -w app
+```
+
+using `yarn workspace`
+
```

Comment:
If we are using `yarn`, maybe we could think about ignoring the `package-lock.json` file because it could get committed in a PR 🤔

Reply:
I've removed all NPM examples